suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -141,7 +143,20 @@ export const prepareTransactionsReducer = createReducerWithExtraDeps(
 
 export const selectIsLoadingTransactions = (state: TransactionsRootState) =>
     state.wallet.transactions.isLoading;
-export const selectTransactionsError = (state: TransactionsRootState) =>
-    state.wallet.transactions.error;
 export const selectTransactions = (state: TransactionsRootState) =>
     state.wallet.transactions.transactions;
+
+// Mobile only selectors
+// These selectors are only used in suite-native package now
```

Comment:
So they can be used in desktop Suite? Because if they can't shouldn't they be passed as extras?

Reply:
They can be used in suite, but some of them have specific behavior that will currently only be used for mobile app. For example the 'selectAccountName' is commented to not be used in suite, because it has some limitations and wouldn't make sense in suite. Once there is local storage, we could refactor it to be more reusable. Other stuff can be used elsewhere.


https://github.com/trezor/trezor-suite/pull/6199/commits/d21956f972b3315afd46be7083fef9311337b8ef